README.md

```diff
@@ -25,7 +25,7 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 *RevenueCat* is the client for the [RevenueCat](https://www.revenuecat.com/) subscription and purchase tracking system. It's 100% `Swift` and compatible with `Objective-C`.
 
 ## Migrating from Purchases v3
-- See our [Migration guide](https://revenuecat-docs.netlify.app/documentation/revenuecat/v4_api_migration_guide)
+- See our [Migration guide](https://revenuecat.github.io/purchases-ios-docs/migration.html)
```

Comment:
Hmm if we have a migration doc in the future, there might be a conflict with this path... I guess ideally this would be something like `v4_api_migration_guide.html` as it was originally? But we can address that later

Reply:
yeah, agreed, updated